### PR TITLE
Cleanup and maintenance

### PR DESCRIPTION
**Pull request by adrianchiris**

### CODE REVIEW

**Review by rollandf — APPROVED** [review]

File: pkg/drivers/http/client.go [hunk 1]
```diff
@@ -35,11 +35,12 @@ func NewClient(isSecure bool, basicAuth *BasicAuth, cert string) (Client, error)
 	httpClient := &http.Client{Transport: http.DefaultTransport}
 	if isSecure {
 		if cert == "" {
-			/* #nosec */
+			//nolint:gosec
 			httpClient.Transport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
 		} else {
 			caCertPool := x509.NewCertPool()
 			caCertPool.AppendCertsFromPEM([]byte(cert))
+			////nolint:gosec
```
Comment (almaslennikov) [inline pkg/drivers/http/client.go:43]: Does the linter directive work with 4 slashes?

**Review by almaslennikov — APPROVED** [review]
Thank you for the effort! LGTM apart from a couple small comments

File: .golangci.yml [hunk 2]
```diff
@@ -1,102 +1,146 @@
...
+  funlen:
+    lines: 120
+    statements: 55
+  goconst:
+    min-len: 2
+    min-occurrences: 2
+  gocyclo:
+    min-complexity: 30
+  goimports:
+    local-prefixes: github.com/Mellanox/maintenance-operator
```
[the remark this answers is not in the document]
Comment (adrianchiris) [reply .golangci.yml:120]: good catch !

File: pkg/drivers/http/client.go (hunk 1 above)
Comment (adrianchiris) [reply pkg/drivers/http/client.go:43]: well it works :) but i will stick to 2.

**Pull request merged**